plugins/confluent/credentials.go

```diff
@@ -0,0 +1,94 @@
+package confluent
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+const (
+	MDS_URL                      = sdk.FieldName("MDS URL")
+	ConfluentCloudCredentials    = sdk.CredentialName("Confluent Cloud Credentials")
+	ConfluentPlatformCredentials = sdk.CredentialName("Confluent Platform Credentials")
```

Comment:
I propose we move these to `credentials/names.go`, and exclude the name of the platform. I believe that keeping both will result in the double appearance of the platform name within any imported credential.

Reply:
Are you talking moving them into the root version of that file or creating my own version within the plugin? I didn't add it to the root because none of the others appeared to be doing so. So figured it wasn't allowed.

Reply:
I just removed these and replaced them with the generic Credential for now.